docs/references/javascript/user/create-metadata.mdx

```diff
@@ -1,100 +1,463 @@
+
+<Callout type="warning">
+  The social provider that you want to connect to must be enabled in your Clerk settings for this method to work. See the [Social connections (OAuth)](/docs/authentication/configuration/sign-in-sign-up-options#social-connections-o-auth) section to learn more.
+</Callout>
+
 ```typescript
 function createExternalAccount: (params: CreateExternalAccountParams) => Promise<ExternalAccount>;
 ```
 
-Adds an external account for the user. A new [`ExternalAccount`][exacc-ref] will be created and associated with the user.
-
 ### `CreateExternalAccountParams`
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `strategy` | [`OAuthStrategy`](/docs/references/javascript/types/oauth-strategy) | The strategy corresponding to the oauth provider, e.g. `oauth_facebook`, `oauth_github`, etc |
-| `redirectUrl?` | `string` | The URL to redirect back to one the oauth flow has completed successfully or unsuccessfully. |
+| `strategy` | [`OAuthStrategy`](/docs/references/javascript/types/oauth#o-auth-strategy) | The strategy corresponding to the OAuth provider. For example: `oauth_facebook`, `oauth_github`, etc. |
+| `redirectUrl?` | `string` | The URL to redirect back to one the OAuth flow has completed successfully or unsuccessfully. |
 | `additionalScopes?` | `string[]` | Any additional scopes you would like your user to be prompted to approve. |
 
-### Returns
+### `createExternalAccount()` returns
 
 | Type | Description |
 | --- | --- |
-| <code>Promise\<[ExternalAccount][exacc-ref]\></code> | The external account created for the user. |
-
-The initial `state` of the returned `ExternalAccount` will be `unverified`. To initiate the connection with the external provider one should redirect to the `externalAccount.verification.externalVerificationRedirectURL` contained in the result of `createExternalAccount`.
-
-Upon return, one can inspect within the `user.externalAccounts` the entry that corresponds to the requested strategy:
+| `Promise<ExternalAccount>` | This method returns a `Promise` which resolves to the [`ExternalAccount`][exacc-ref] created for the user. |
+
+### `createExternalAccount()` example
+
+After calling `createExternalAccount`, the initial `state` of the returned `ExternalAccount` will be `unverified`. To initiate the connection with the external provider, redirect the user to the `externalAccount.verification.externalVerificationRedirectURL` contained in the result of `createExternalAccount`.
+
+Upon return, inspect within the `user.externalAccounts` the entry that corresponds to the requested strategy:
+
+- If the connection was successful, then `externalAccount.verification.status` should be `verified`.
+- If the connection was not successful, then the `externalAccount.verification.status` should not be `verified` and the `externalAccount.verification.error` will contain the error encountered so that you can present corresponding feedback to the user.
```

Comment:
```suggestion
- If the connection was not successful, then the `externalAccount.verification.status` will be `unverified` and the `externalAccount.verification.error` will contain the error encountered, which you can present to the user.
```

Reply:
I think the status will be `failed` if the verification fails. the status can also be `unverified`.

<img width="680" alt="Screenshot 2024-03-18 at 17 54 58" src="https://github.com/clerk/clerk-docs/assets/98043211/7d5c09ed-8eff-499e-8cef-16e2794c2161">

Reply:
Got it. Your original wording is the most accurate way to describe it I think. Although I prefer "will not be" over "should not be", as it leaves less room for confusion. I assume it's not expected behavior that an unsuccessful connection will sometimes be `verified`. If that is expected, we'll need to explain why.